crates/apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -380,29 +383,16 @@ where
                                 .accept(tx_index);
                         }
                         // events from other sources
-                        response.events.extend(
-                            // ibc events
-                            result
-                                .ibc_events
-                                .iter()
-                                .cloned()
-                                .map(|ibc_event| {
-                                    ibc_event.with(Height(height)).into()
```

Comment:
Hermes requires the height in IBC events. We have to add `height` in another part if removing here.

Reply:
we do add it yeah